v2x/vlog_to_pbtype.py

```diff
@@ -345,12 +377,12 @@ def pin_sort(p):
 
 
 def mode_interconnects(mod, mode_name) -> List[(CellPin)]:
-    interconn = []
+    interconn = {}
```

Comment:
I don't think a dictionary for interconn works as there are potential multiple connects from a single source?

Reply:
But the dictionary is indexed with drivers and contains lists of sinks. So the situation when there are multiple sinks per driver is supported.

Reply:
Can you add comments to this to describe how this works.

Reply:
Ok, I've added a docstring to the function.